src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const bodyElement = document.body;
+const spanElement = bodyElement.querySelectorAll('.population');
+let total = 0;
+const arr = [];
```

Comment:
what does this variable store?
it should be more meaningful name for variable, not just `arr`